languages/translations/en.json

```diff
@@ -166,17 +166,20 @@
             "default": {
                 "default": "Turn {modifier}",
                 "name": "Turn {modifier} onto {way_name}",
-                "destination": "Turn {modifier} towards {destination}"
+                "destination": "Turn {modifier} towards {destination}",
+                "junction_name": "Turn {modifier} at {junction_name}"
             },
             "straight": {
                 "default": "Continue straight",
                 "name": "Continue straight onto {way_name}",
-                "destination": "Continue straight towards {destination}"
+                "destination": "Continue straight towards {destination}",
+                "junction_name": "Continue straight at {junction_name}"
             },
             "uturn": {
                 "default": "Make a U-turn at the end of the road",
                 "name": "Make a U-turn onto {way_name} at the end of the road",
-                "destination": "Make a U-turn towards {destination} at the end of the road"
+                "destination": "Make a U-turn towards {destination} at the end of the road",
+                "junction_name": "Make a U-turn at {junction_name} at the end of the road"
```

Comment:
`end of road` instructions were introduced as a user convenience, so that the user would know they didn’t need to keep track of the remaining distance and could focus on reaching the end of the road. It may be desirable to avoid overloading the user with both the junction name and the “end of road” phrase.

Reply:
Agree, updated in https://github.com/Project-OSRM/osrm-text-instructions/pull/291/commits/a4d24234d91f06295eb882900c334eac34b52f35.